README.md

```diff
@@ -56,7 +56,8 @@ There is an explanation on what fields to use with what kind of contents, as wel
  custom_cut3        | Third custom demographic field of  the individual (unique to the organization)  | Text, Max length 255                                           | Yes, no                                                                        | False                                | False                                |                              
  custom_cut4        | Fourth custom demographic field of  the individual (unique to the organization) | Text, Max length 255                                           | 31Wing                                                                         | False                                | False                                |                              
  custom_cut5        | Fifth custom demographic field of  the individual (unique to the organization)  | Text, Max length 255                                           | Could be anything                                                              | False                                | False                                |                              
- sso                | Indicates whether the user should be accessing by SSO or not                    | Text, Max length 255                                           | Could be anything                                                              | False                                | False                                |                              
+ sso                | Indicates whether the user should be accessing by SSO or not                    | Text, Max length 255                                           | Could be anything                                                              | False                                | False                                |
+ otp                | One time password, provided by the employer, which has to be confirmed by user  | Text, Max length 36                                            | abcABC123                                                                      | False                                | False                                | Only letters and numbers, no special characters                            
```

Comment:
Wasn't there any mention about dash `-` or am I mistaken? `{company-specific-code}-{actual-code}`

Reply:
I don't remember, I'll ask for clarification